src/pages/home/HomePage.js

```diff
@@ -322,7 +321,9 @@ class App extends React.Component {
                                         shouldShowNavigationMenuButton={this.state.isSmallScreenWidth}
                                         onNavigationMenuButtonClicked={this.toggleNavigationMenu}
                                     />
-                                    {this.props.currentURL === '/settings' && <SettingsPage />}
+                                    <SettingsModal
+                                        isVisible={this.props.currentURL === '/settings'}
```

Comment:
This was left by someone else but can we take this opportunity to update `/settings` to use the constant in `ROUTES`?